packages/cfd/src/features/Containers/cfd-trade-modal.tsx

```diff
@@ -29,7 +29,7 @@ const CFDTradeModal = observer(
         const { platform } = common;
         const { is_mobile } = ui;
 
-        const { mt5_trade_account, dxtrade_tokens, ctrader_tokens } = useCfdStore();
+        const { mt5_trade_account, dxtrade_tokens, ctrader_tokens, loadCTraderTokens } = useCfdStore();
```

Comment:
This part will be removed completely as it was added based on our old refactorings.